docs/release-notes/2.41.1.md

```diff
@@ -26,7 +26,7 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 
 
 ## Improvements
-
+- [PMM-12712](https://jira.percona.com/browse/PMM-12712) - A new metric collection, `mongodb_sharded_collection_chunks_count`, has been introduced to show how the collection is distributed among different shards.on distribution across different shards. 
```

Comment:
```suggestion
- [PMM-12712](https://jira.percona.com/browse/PMM-12712) -  Added a new metric collection for MongoDB exporters: `mongodb_sharded_collection_chunks_count`. Use this new metric to assess the distribution of a collection across different shards, and get valuable insights into their load balance.
```

Reply:
Name of metric were change to 'mongodb_shards_collection_chunks_count' so instead _sharded_ there is _shards_. It would be good to mention that metric is for sharded Mongos. Everything else sounds good.